log4j-gctests/src/test/java/org/apache/logging/log4j/gctests/GcFreeMixedSyncAyncLoggingTest.java

```diff
@@ -29,7 +29,7 @@
 @Tag("allocation")
 @Tag("functional")
 @Category(GarbageFree.class)
-public class GcFreeMixedSyncAyncLoggingTest {
+public class GcFreeMixedSyncAsyncLoggingTest {
```

Comment:
Sigh, java classes require a rename -- I have a script to do that but generally forget to use it since most of the time I'm not touching java projects...